src/Resources/Locales/en.json

```diff
@@ -583,6 +583,61 @@
             "libraryName": "Panel",
             "description": "A layout panel"
         },
+        "dataHistory": {
+            "title": "Data history widget",
+            "libraryName": "Data history",
+            "description": "Show timeseries data in context of your 3D scene",
+            "notEnabledDescription": "Data history is not enabled for this ADT instance.",
+            "form": {
+                "connectionStringTitle": "Connection string",
+                "connectionStringPlaceholder": "Enter connection string (e.g. key1=value1;key2=value2;key3=value3)",
+                "connectionStringInformation": "Data history widgets target the Azure Data Explorer instance connected to the Azure Digital Twins instance at the time of creation.",
+                "connectionLoadingText": "Loading connection information...",
+                "noConnectionInformationText": "No connection information found",
+                "displayNamePlaceholder": "Enter display name",
+                "timeSeries": {
+                    "timeSeriesDescription": "Add up to {{maxTimeSeriesCount}} timeseries to display",
+                    "title": "Time series",
+                    "description": "A series to display in the line chart",
+                    "property": "Property",
+                    "unit": "Unit",
+                    "label": "Label",
+                    "labelPlaceholder": "Enter label for this series to show in chart",
```

Comment:
This value is too long to show. We should shorten it so it is visible in the field.
Maybe "Enter label to show in chart legend"
![image](https://user-images.githubusercontent.com/57726991/195202275-0b9bd268-1c07-489f-b75f-fe2373f6b6ad.png)

Reply:
since it is not just legend, I just changed it to "Enter label to show in chart"

Reply:
we don't usually say where it's used, I'd almost think it's better to just go with "Enter series label". I don't feel strongly so feel free to resolve if you disagree.